adapters/zwave/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
When you require a directory, it loads the index.js file from the directory.
See https://nodejs.org/api/modules.html#modules_all_together for the full set of loading rules used by require.